src/Microsoft.ML.FastTree/GamClassification.cs

```diff
@@ -171,7 +171,7 @@ private protected override SchemaShape.Column[] GetOutputColumnsCore(SchemaShape
     /// <summary>
     /// The model parameters class for Binary Classification GAMs
     /// </summary>
-    public sealed class BinaryClassificationGamModelParameters : GamModelParametersBase, IPredictorProducing<float>
+    public sealed class GamBinaryModelParameters : GamModelParametersBase, IPredictorProducing<float>
```

Comment:
`BinaryClassification`?

Do we need to put `Classification` in the name? We do everywhere else.

Reply:
For ModelParameters we do not.  Please see the other comment.  

---
In reply to: [266626783](https://github.com/dotnet/machinelearning/pull/2968#discussion_r266626783) [](ancestors = 266626783)